docs/docs/how-to/testing/visual-testing-with-storybook.md

```diff
@@ -83,22 +83,32 @@ The final `.storybook/main.js` should look something like this:
 
 ```js:title=.storybook/main.js
 module.exports = {
-  // You will want to change this to wherever your Stories will live
-  stories: ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
-  addons: ["@storybook/addon-links", "@storybook/addon-essentials"],
+  "stories": [
+    "../src/**/*.stories.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    //"storybook-addon-gatsby"
+  ],
+  "framework": "@storybook/react",
   // highlight-start
-  core: {
-    builder: "webpack5",
+  "core": {
+    "builder": "webpack5"
   },
   webpackFinal: async config => {
     // Transpile Gatsby module because Gatsby includes un-transpiled ES6 code.
     config.module.rules[0].exclude = [/node_modules\/(?!(gatsby)\/)/]
```

Comment:
According to https://github.com/prismicio-community/storybook-addon-gatsby/pull/8 this also needs a change?